scania-digital-design-system/sdds
# Feature/typography package

**Describe pull-request**  
- Add function to generate typography scale and type sets

**Solving issue**  
_Add which issue this pull-request solves by adding # plus the number of the issue (for example #123)_
Fixes: https://github.com/scania/corporate-ui/issues/591

**How to test**  
Run storybook inside components and see the typography section
Compare with design in Figma
